OpenTESArena/src/Entities/Doodad.h

```diff
@@ -12,8 +12,8 @@
 class Doodad : public Entity
 {
 private:
-	Animation animation;
 	Double3 position;
+	Animation animation;
```

Comment:
I guess this change is okay. I normally put larger structs first, but that's only really relevant to byte padding which is hardly a concern here.